lib/pe_build/provisioner/pe_agent.rb

```diff
@@ -139,6 +139,18 @@ def provision_pe_repo
       def provision_posix_agent
         shell_config = Vagrant.plugin('2').manager.provisioner_configs[:shell].new
         shell_config.privileged = true
+
+        certname_option = case config.certname
+                          when 'hostname'
+                            "agent:certname=#{machine.config.vm.hostname}"
+                          when 'fqdn'
+                            #The installer script already defaults to FQDN
+                            #Just let it do its thing
+                            ''
+                          when 'vm_name'
+                            "agent:certname=#{machine.name}"
+                          end
```

Comment:
There's no default statement, so if the provided value doesn't match then this will return nil. Is this intended?